cmd/controller-bootstrap/command/root.go

```diff
@@ -43,31 +44,35 @@ var rootCmd = &cobra.Command{
 }
 
 func init() {
-	rootCmd.PersistentFlags().StringVarP(
-		&optServiceAlias, "aws-service-alias", "s", "", "AWS service alias",
+	rootCmd.PersistentFlags().StringVar(
+		&optServiceAlias, "aws-service-alias", "", "AWS service alias",
 	)
-	rootCmd.PersistentFlags().StringVarP(
-		&optRuntimeVersion, "ack-runtime-version", "r", "", "Version of aws-controllers-k8s/runtime",
+	rootCmd.PersistentFlags().StringVar(
+		&optRuntimeVersion, "ack-runtime-version", "", "Version of aws-controllers-k8s/runtime",
 	)
-	rootCmd.PersistentFlags().StringVarP(
-		&optAWSSDKGoVersion, "aws-sdk-go-version", "v", "", "Version of github.com/aws/aws-sdk-go used to infer service metadata and resources",
+	rootCmd.PersistentFlags().StringVar(
+		&optAWSSDKGoVersion, "aws-sdk-go-version", "", "Version of github.com/aws/aws-sdk-go used to infer service metadata and resources",
 	)
-	rootCmd.PersistentFlags().BoolVarP(
-		&optDryRun, "dry-run", "d", false, "Optional: if true, output files to stdout",
+	rootCmd.PersistentFlags().BoolVar(
+		&optDryRun, "dry-run", false, "Optional: if true, output files to stdout",
 	)
-	rootCmd.PersistentFlags().BoolVarP(
-		&optExistingController, "existing-controller", "e", false, "Optional: if true, update the existing service controller",
+	rootCmd.PersistentFlags().BoolVar(
+		&optExistingController, "existing-controller", false, "Optional: if true, update the existing service controller",
```

Comment:
Does this need to be a flag, even? We can check that the files already exists and run the different routine conditionally? The code shouldn't do anything destructive, anyway, so even if someone runs this on an existing repository it won't ruin anything.